scripts/watch-from-dependency.js

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
+
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+"use strict";
+
+const cp = require("child_process");
+const cla = require("command-line-args");
+const path = require("path");
+const fs = require("fs");
+
+const realmPackagePath = path.resolve(__dirname, "..");
+
+const TRIGGERS = [{
+  path: "src",
+  pattern: "src/*",
+}, {
+  path: "lib",
+  pattern: "lib/*",
+}, {
+  path: "react-native",
+  pattern: "react-native/*",
+}];
```

Comment:
Add `/types` to watchlist.

```suggestion
}, {
  path: "types",
  pattern: "types/*",
}];
```